modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,5 @@
+.xref:index.adoc[Government Cloud]
+* xref:index.adoc[Government Cloud]
+* xref:gov-cloud-features.adoc[Features Included in Government Cloud]
+* xref:gov-cloud-security.adoc[Security in Government Cloud]
+* xref:gov-cloud-account-setup.adoc[Set Up Your Government Cloud Account]
```

Comment:
Guessing that we do this for SEO reasons…but it feels like it's a lot to end every article in "Government Cloud". I think it would be nice to omit it.

Reply:
Yep, SEO and context when they land there from search results. And consistency with the other deployment "flavors" like EU Control Plane.